awsx/ec2/vpc.ts

```diff
@@ -161,7 +157,7 @@ export class Vpc extends schema.Vpc<VpcData> {
               mapPublicIpOnLaunch: spec.type.toLowerCase() === "public",
               cidrBlock: spec.cidrBlock,
               tags: {
-                ...args.tags,
```

Comment:
This change has no impact on behaviour. The only difference between `args.tags` and `sharedTags` is adding a Name, but the name will always be overwritten below.

Reply:
that's true, it was more for consistency/readability. No need to think about "why use `arg.tags` here instead of `sharedTags`